# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1396731795 14400
#      Sat Apr 05 17:03:15 2014 -0400
# Branch stable
# Node ID 6fdd3ab55b7823b4291e5f21f5f3dad21229cf43
# Parent  c644ed73c6cec0af602641cd53f8f8e70887db06
undo unintended changes to fplot in changeset 37c300acfcfd

diff --git a/scripts/plot/draw/fplot.m b/scripts/plot/draw/fplot.m
--- a/scripts/plot/draw/fplot.m
+++ b/scripts/plot/draw/fplot.m
@@ -141,20 +141,20 @@ function [X, Y] = fplot (varargin)
   ## FIXME: This algorithm should really use adaptive scaling as the 
   ##        the numerical quadrature algorithms do so that extra points are
   ##        used where they are needed and not spread evenly over the entire
   ##        x-range.  Try any function with a discontinuity, such as
   ##        fplot (@tan, [-2, 2]) or fplot ("1./x", [-3, 2]), to see the
   ##        problems with the current solution.
 
   while (n < 2^18)    # Something is wrong if we need more than 250K points
-    yi = interp1 (x0, y0, x, "linear")
+    yi = interp1 (x0, y0, x, "linear");
     ## relative error calculation using average of [yi,y] as reference
     ## since neither estimate is known a priori to be better than the other.
-    err = 0.5 * max (abs ((yi - y) ./ (yi + y))(:))
+    err = 0.5 * max (abs ((yi - y) ./ (yi + y))(:));
     if (err < tol || abs (err - err0) < tol/2)
       ## Either relative tolerance has been met OR
       ## algorithm has stopped making any reasonable progress per iteration.
       break;
     endif
     x0 = x;
     y0 = y;
     err0 = err;
